parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -158,6 +158,7 @@ $global:AzureCNIConfDir = [Io.path]::Combine("$global:AzureCNIDir", "netconf")
 $global:NetworkPlugin = "{{GetParameter "networkPlugin"}}"
 $global:VNetCNIPluginsURL = "{{GetParameter "vnetCniWindowsPluginsURL"}}"
 $global:IsDualStackEnabled = {{if IsIPv6DualStackFeatureEnabled}}$true{{else}}$false{{end}}
+$global:IsAzureCNIOverlayEnabled = {{if IsAzureCNIOverlayFeatureEnabled}}$true{{else}}$false{{end}}

```

Comment:
1. Need to run `make generate` to update templates
2. The changes in staging/cse/windows need to publish a new cse package so we need to follow below guidance to check in the changes. Otherwise it will fail when this cse change is vendored to rp with old cse package
    a. Update CSE script package in staging/cse/windows
    b. Publish a new CSE script package after above PR is merged
    c. Update parts/windows/kuberneteswindowssetup.ps1 with your changes and the new CSE script package version `$WindowsCSEScriptsPackage`.

Reply:
so we'll split this PR into two parts like below?
1. staging/cse/windows
2. parts/windows & pkg/agent

Reply:
yes. In "2. parts/windows & pkg/agent", you need a new cse script package

Reply:
I've split the PR into the 2 parts, part 2 -> #2408